database/index.js

```diff
@@ -0,0 +1,31 @@
+var mysql = require('mysql');
+var config = require('../config/config.js');
+var db = mysql.createConnection(config);
+
+db.connect(function(error) {
+  if (error) {
+    console.log(error);
+  } else {
+    console.log('==mySQL is connected==');
+  }
+});
+
+const getPhotos = function (experienceID, callback) {
+  var id = experienceID
```

Comment:
you created an id variable on line 14, but didn't use it on line 15.  you can probably remove this one